package com.example.esercizio_3_restful.exception;public class CentralizedExceptionHandler {
}
